src/typed/env.mli

```diff
@@ -96,16 +97,15 @@ val add_local_type: Path.t -> type_declaration -> t -> t
 val add_item: signature_item -> t -> t
 val add_signature: signature -> t -> t
 
-(* Remember the name of the current compilation unit. *)
-val set_unit_name: string -> unit
-val get_unit_name: unit -> string
+(* Remember the current compilation unit. *)
+val set_unit: string * string -> unit
```

Comment:
@ospencer Can you explain this one?

Reply:
The name alone isn't enough to determine if we're in the same module, since modules could have the same name. This makes it filepath * name instead.

Reply:
Can you maybe add a comment to this explaining what the two tuple elements are?